### PR TITLE
Don't log search queries

### PR DESCRIPTION
**Pull request by gmethvin**
I'd like to start archiving the logs in S3 so we should make sure private information like this isn't logged in production.

### CODE REVIEW

File: shoebox/app/com/keepit/controllers/ext/ExtSearchController.scala [hunk 1]
```diff
@@ -63,7 +63,7 @@ class ExtSearchController @Inject() (
     }
 
     val userId = request.userId
-    log.info("searching with %s using userId id %s".format(query, userId))
+    log.debug(s"""searching for "$query" with userId $userId""")
```
Comment (ymatsuda) [inline shoebox/app/com/keepit/controllers/ext/ExtSearchController.scala]: I have used this logging to make sure the search server was getting traffic. I want an alternative to this logging. Maybe real time event streaming with some enhancement?

Comment (2is10) [inline shoebox/app/com/keepit/controllers/ext/ExtSearchController.scala]: or just log the user ID and the length of the query (num chars)?

Comment (gmethvin) [inline shoebox/app/com/keepit/controllers/ext/ExtSearchController.scala]: Good question. We could persist search events to mongo and make an admin event page similar to the one @andrewconner made (at https://keepitfindit.com/admin/realtime/event)

I'm also fine with just logging that we got a search request, maybe with a userid, if that's all you need. My main concern is that I don't want to store logs that contain all our users' private search history.

Comment (gmethvin) [inline shoebox/app/com/keepit/controllers/ext/ExtSearchController.scala]: I'm okay with doing Jared's suggestion (logging the length of the query and user id).

**Comment by gmethvin** [comment]
I updated this to use Jared's suggestion of just logging the length of the query. I'm also open to discussing other monitoring tools to let us see queries as they come in.

**Comment by 2is10** [comment]
We could send them to SnapChat on our phones. Ha!

**Pull request merged**